content/docs/design/dynamic-concurrency/dynamic-concurrency.md

```diff
@@ -80,7 +80,8 @@ Job-a:
 
 # **get_result  function** 
 
-get_result (job-target,Split)
+* getresult is only available in the vars_iter field.
+* get_result (job-target,Split)
```

Comment:
note: the `job-target` should be consistent with line 99

Reply:
changed